apps/fz_http/test/fz_http/repo/notifier_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule FzHttp.Repo.NotifierTest do
+  use ExUnit.Case, async: false
+
+  import FzHttp.TestHelpers
+
+  alias Ecto.Adapters.SQL.Sandbox
+  alias FzHttp.Events
+  alias FzHttp.Repo
+
+  setup do
+    start_supervised!({Postgrex.Notifications, [name: Repo.Notifications] ++ Repo.config()})
+    start_supervised!(Repo.Notifier)
+    :ok = Sandbox.checkout(Repo)
+  end
+
+  test "adds and removes user from wall state on db notifications" do
```

Comment:
Might it be better to unit test add and remove separately?

Reply:
I was also getting db ownership errors here when splitting them up, but I'll take a look again. I'd really like to get them into a more standard form!

Reply:
Ahh I see. Yeah those can be hard to track down. Usually `async: false` fixes it but maybe some process was lingering around after these finished.